index.html

```diff
@@ -33,27 +33,27 @@ <h3>I'am a Software Developer</h3>
           class="main-icons"
           src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio-\Icons\Git-hub-icon.png"
           alt="git-hub icon"
-        >
+        />
         <img
           class="main-icons"
-          src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio-\Icons\Icon - Linkedin.png"
+          src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio\Icons\Icon - Linkedin.png"
           alt="instagram icon"
-        >
+        />
         <img
           class="main-icons"
           src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio-\Icons\Icont number 3.png"
           alt="new icon"
-        >
+        />
         <img
           class="main-icons"
           src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio-\Icons\Twitter icon.png"
           alt="twitter icon"
-        >
+        />
         <img
           class="main-icons"
           src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio-\Icons\Microverse Icon.png"
           alt="microverse icon"
-        >
+        />
```

Comment:
The list of icons should use a `ul` tag to group them, and `li` per icon. Each icon should be a link (`a` tag) because it must be clickable in order to redirect you to the clicked social media website.

The structure should look more or less like like:
```
      <ul class="container-icons">
        <li>
          <a href="#">
            <img
              class="main-icons"
              src="./Icons/Git-hub-icon.png"
              alt="git-hub icon"
            />
          </a>
        </li>
        <li>
          <a href="#">
            <img
              class="main-icons"
              src="./Icons/Icon - Linkedin.png"
              alt="instagram icon"
            />
          </a>
        </li>
        <li>
          <a href="#">
            <img
              class="main-icons"
              src="./Icons/Icont number 3.png"
              alt="new icon"
            />
          </a>
        </li>
        <li>
          <a href="#">
            <img
              class="main-icons"
              src="./Icons/Twitter icon.png"
              alt="twitter icon"
            />
          </a>
        </li>
        <li>
          <a href="#">
            <img
              class="main-icons"
              src="./Icons/Microverse Icon.png"
              alt="microverse icon"
            />
          </a>
        </li>
      </ul>
```